Resources/Locale/en-US/sticky/sticky-component.ftl

```diff
@@ -7,3 +7,8 @@ comp-sticky-success-unstick-bomb = You removed the bomb
 
 # General strings
 comp-sticky-unstick-verb-text = Unstick
+
+#split stack fail message
+cannot-split-due-to-sticky = It is too sticky !
```

Comment:
Please remove here and in code any redundant whitespaces before punctuation marks.